mscore/editstyle.ui

```diff
@@ -6,14 +6,29 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>1150</width>
-    <height>739</height>
+    <width>780</width>
+    <height>578</height>
    </rect>
   </property>
+  <property name="sizePolicy">
```

Comment:
Without a single change to the original UI, the "remember last user resize" and "remember last user position" still works,
But the window can only be expanded. So all these changes are to allow the window to contract. But sure, will go through each one and see what is absolutely necessary.